addons/account_voucher/migrations/9.0.1.0/pre-migration.py

```diff
@@ -10,8 +10,18 @@
 }
 
 
+def delete_payment_views(cr):
```

Comment:
This view was calling a failure during testing, because it referred to no longer existing fields, during upgrade of the newly migrated database.